### ISSUE TITLE
CPU/GPU Spec you've used for running main_bayesian.py

### ISSUE THREAD

**Issue by YeonghyeonKO** [kumar-shridhar/PyTorch-BayesianCNN#61]
Hi, I'm studying Bayesian CNN theseday,

and I've tried to run `main_bayesian.py` with MacBook Pro ('16 2019 16GB Ram)

**n_epochs** and **batch_size** (the hyper-parameter you've set) are 200 and 256 each,

but I'm afraid my MacBook cannot withstand the load for training BCNN.

So, I changed the number of epochs and batches to 10 and 100, respectively.

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/46114393/111250602-f6bcb500-8650-11eb-9b5f-6d462196fcdd.png">

As the above photo, Validation Accuracy exceeds 0.97 (I think it's not bad)

I'm curious about your H/W Spec when designing and testing this model you've uploaded.

**Comment by kumar-shridhar** [comment]
Hi, 
I did not benchmark the Hardware specs but you can run this on Google Colab with free GPUs there. 
In general my assumption will be Bayesian CNN will take twice the computation needed for non-Bayesian methods. This is primarily due the computation of both means and variances.

**Comment by YeonghyeonKO** [comment]
Thank you for answering me :)
